app/src/js/utils/format.js

```diff
@@ -302,6 +302,7 @@ export const providerLink = (provider) => {
 export const bool = (value) => (value ? 'Yes' : 'No');
 
 export const displayCase = (string) => {
+  if (!string) return nullValue;
```

Comment:
Does this fix a bug?

Reply:
I would say "yes" and  making sure if something comes up undefined or empty as a check for the link. I saw it throughout this file so I made sure it was here too.